Widgets/Bars/ClassBar.lua

```diff
@@ -493,7 +499,7 @@ local function Update(button, event)
 end
 
 ---@param self ClassBarWidget
-local function Enable(self)
+local function UpdateEventListeners(self)
```

Comment:
If you want to make a function for updating listeners, then that function should only update the listeners.

The initial logic in this function should remain inside of `Enable()` because it's static information, that we don't need to update.

The biggest problem with this change is that the `false` return on the `Enable` call has now been abstracted away.

Reply:
Good point.